recipes/fugue/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "fugue" %}
+{% set version = "0.6.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fugue-{{ version }}.tar.gz
+  sha256: 602e69cea45a600a1f6ae2011c115ad25a5608cb25d78717da54edc6e8162d23
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - adagio >=0.2.3
+    - pandas >=1.0.2
+    - pyarrow >=0.15.1
+    - python >=3.6
+    - qpd >=0.2.4
+    - sqlalchemy
+    - triad >=0.5.4
+
+test:
+  imports:
+    - fugue
+    - fugue._utils
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://github.com/fugue-project/fugue
+  summary: An abstraction layer for distributed computation
+  license: Apache-2.0
+  license_file: LICENSE
```

Comment:
Please add a comment about the `license_file` line:

```python
# license file manually packaged. See: https://github.com/fugue-project/fugue/pull/245
```

This will be a visual reminder to check to see if the license file is included in future releases, at which point you can remove the manually packaged copy.

Reply:
Got it, I have added that